scripts/ci/junit2slack/main.go

```diff
@@ -0,0 +1,152 @@
+package main
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"github.com/GoogleCloudPlatform/testgrid/metadata/junit"
+	"github.com/slack-go/slack"
+	"log"
+	"os"
+)
+
+func main() {
+	// We're only logging errors currently. If we log more, we should probably switch logging libraries
+	log.SetOutput(os.Stderr)
+
+	var slackAttachments []slack.Attachment
+	var junitFiles []*junit.Suites
+
+	// We should accept all file names at once since we're using `go run` to run this program. No need to recompile
+	// for each file we want to parse
+	for _, fileName := range os.Args[1:] {
+		if _, err := os.Stat(fileName); err == nil {
+			data, err := os.ReadFile(fileName)
+			if err != nil {
+				log.Printf("error while reading %s: %s", fileName, err)
+				continue
+			}
+
+			junitSuites, err := junit.Parse(data)
+			if err != nil {
+				log.Printf("error while parsing junit suites in %s: %s", fileName, err)
+				continue
+			}
+			junitFiles = append(junitFiles, junitSuites)
+
+		} else if errors.Is(err, os.ErrNotExist) {
+			log.Printf("%s doesn't exist: %s", fileName, err)
```

Comment:
one annoying thing about `Printf` if it does not add `\n`, so we might want to add that ourselves

Reply:
Looks like each `Logger` method will end up using [Logger.Output](https://pkg.go.dev/log#Logger.Output) which says:

> A newline is appended if the last character of s is not already a newline.

That being said, I'm down to change this if we do something similar elsewhere in the repo